recipes/mykatlas/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: mykatlas
+  version: "0.6.1"
+
+source:
+  fn: mykatlas-0.6.1.tar.gz
+  url: https://pypi.python.org/packages/bf/33/946e43324bc24ba331a1bee06e609832bfbe7a4f07dde1846246fb5a68cf/mykatlas-0.6.1.tar.gz
+  md5: 87d59770ceeb5406a0146c3c51d93ea9
+
+build:
+  number: 0
```

Comment:
Please add the following to the `build:` section to get entry points created by Conda:
```yaml
  entry_points:
    - atlas = mykatlas.atlas_main:main
```

Reply:
Fixed